### PR TITLE
044 testing test task service

### PR DESCRIPTION
**Pull request by zoroksana**

### CODE REVIEW

File: components/courses/tmp/rspec_guard_result [hunk 1]
```diff
@@ -0,0 +1 @@
+0 examples, 0 failures in 0.0004 seconds
```
Comment (sveredyuk) [inline components/courses/tmp/rspec_guard_result]: remove

File: components/courses/spec/services/courses/test_task/create.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Courses::TestTask::Create do
+  let!(:season)          { create(:season, title: 'Test Season') }
+  let!(:user)            { create(:user) }
+  let!(:student)         { ::Courses::Student.create(season_id: 1, user_id: 1, status: :enrolled,
```
Comment (sveredyuk) [inline components/courses/spec/services/courses/test_task/create.rb:6]: ::Courses::Student.create(season_id: season.id, user_id: user.id....

Comment (denys-medynskyi) [reply components/courses/spec/services/courses/test_task/create.rb:6]: What about creating a factory girl?
Define factory and use `create(:student)`, etc.

Comment (sveredyuk) [reply components/courses/spec/services/courses/test_task/create.rb:6]: @denysmedynskyi I will fix it myself. We need asap finish dev today and open huge PR into development

**Pull request closed without merging**